### ISSUE TITLE
Window does not open

### ISSUE THREAD

**Issue by eng3** [Vulnerator/Vulnerator#157]
#### Prerequisites
_Before submitting a new issue, please ensure you have completed the following (delete the space in the box and replace it with an "x" to denote that it has been completed)_

- [x] I have ensured that I am running the latest release
- [x] The issue is repeatable
- [x] The issue has not already been reported

#### Description
No window appears after running Vulnerator.  It does appear in task manager

#### To Reproduce
1. Unzip files
2. Double Click on Vulnerator

#### Expected Outcome
Window should appear

#### Actual Outcome
Nothing appears

#### System Information
Windows 7 SP1

#### Additional Context
System is on an isolated network

**Comment by eng3** [comment]
The symptoms sound similar with the exception that in #152, he says the the process terminates immediately.  In my case, the process keeps running indefinitely until I kill it in task manager.  Is there a way to get some debug info out so you have more to go on?

**Comment by BillCosner** [comment]
Same error for me. Win 10 1703. Isolated network. Multiple computers. 

Event Log shows 2 errors:
1) Unhandled exception in .NET 4.0.303119 
2) application error in module KERNELBASE.dll ver 10.0.15063.1266

**Comment by eng3** [comment]
I do not see any event log messages

**Comment by amkuchta** [comment]
@eng3 @BillCosner do you by any chance have HIPS installed?  If yes, could you double check that nothing is being blocked there?

**Comment by BillCosner** [comment]
@amkuchta Yes, HIPS is installed, but I checked the logs and had the machine put into tuning mode. Nothing was blocked. I also looked for the Vulnerator folder in my Appdata folder, but nothing named Vulnerator was there. I also checked with my ISSM who reported the issue, and he has no Vulnerator folder in his AppData/Roaming folder either.

**Comment by BillCosner** [comment]
Sorry to hijack this thread. Can it be merged with the other thread which seems more connected?

**Comment by amkuchta** [comment]
@BillCosner I can't "merge" the threads, per se, but I  can close this one and provide a link to the other thread, where we can continue the discussion.

@eng3, thoughts on that?  Also, looking through the debug file you sent me, it appears that SQLite is failing to run on your system, which means that it looks like either the `*.dll` files are missing or something is blocking them.  Can you run a search through the Vulnerator folder for `*sqlite*`, please, and let me know what comes back?

**Comment by BillCosner** [comment]
@amkuchta Please do. I had a chance to troubleshoot today. I can open it fine with my desktop admin account but not with my standard user account.

**Comment by eng3** [comment]
@BillCosner , I suggest that you just open a new issue since your issue seems to have completely different symptoms.  For example, mine does not run as admin, I DO have a vulnerator folder in appdata/roaming, I dont get error messages, I am using a different OS.

@amkuchta  Which vulnerator folder did you want me to search for sqlite?  I already sent you what's in that folder (Findings.sqlite, VulneratorDatabase.sqlite) but they are empty.  If there was a DLL file, wouldn't vulnerator fail to start he process because of missing symbols?  From the error message, it seems more like the sqlite functions are properly loaded into memory but the execution is failing returning the "cannot open database" message.  Is there any way to get further debug output from the sqlite function that is returning the error?

**Comment by amkuchta** [comment]
@eng3 I apologize, I should have been more specific - the DLL would be found in the Vulnerator file you run the application from. As for it being missing vs. blocked, it could be either one - I had to work some magic with SQLite specifically to get it to run on NMCI, which meant packaging it as a local file instead of how it typically runs within the application. Because of this, the application will still run without it, but if it doesn't find the library, it errors out. On the flip side of that, you may be right, it could just be being blocked (which is more likely), but I want to go the route of "process of elimination".

**Comment by eng3** [comment]
@amkuchta I see System.data.sqlite.xml, system.data.sqlite.linq.dll, system.data.sqlite.ef6.dll, system.data.sqlite.dll.config, system.data.sqlite.dll, and sqlite.interop.dll in each x86/x64.  Do you know of any GPO setting that would block sqlite?

**Comment by eng3** [comment]
@amkuchta any new ideas?

**Comment by eng3** [comment]
@amkuchta Is there a way to have the database (and other files) created in another folder?

**Comment by amkuchta** [comment]
@eng3 unfortunately, in v6.1.9, the answer to that is no.  I changed that for v6.2.0, but I've been slow in getting the new version out due to other responsibilities at work and at home.

Where are you running the Vulnerator executable from?  That, in my opinion, may be the more likely culprit.

**Comment by eng3** [comment]
I have tried a network share, desktop (another network share), a folder on the c: drive all with the same result.  The userprofile (network share) location that it creates that database files which it cannot open has not changed so I thought that may be the culprit.

**Comment by genolewis** [comment]
> The symptoms sound similar with the exception that in #152, he says the the process terminates immediately. In my case, the process keeps running indefinitely until I kill it in task manager. Is there a way to get some debug info out so you have more to go on?

This is our issue.  The issue i've noticed is either with roaming profiles or folder redirection.  If I login with a local account on the machine then vulnerator runs fine.  We have all folders set to redirection including appdata(roaming).  The application is being ran from a folder on the c drive. 

I may have a chance this week to isolate the issue.
